### PR TITLE
PLATML-2354 Implement default metric evaluation logic for Regression …

### PR DESCRIPTION
**Pull request by marina-bjegovic**
…algorithm

### CODE REVIEW

File: recipes/python/retail/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # from Adobe.
 #####################################################################
 
-FROM adobe/acp-dsw-ml-runtime-python:0.20.1
+FROM docker-ml-runtime-tensorflow-snapshot.dr-uw2.adobeitc.com/marina-ml-runtime-python:0.0.1
```
Comment (marina-bjegovic) [inline recipes/python/retail/Dockerfile]: This line will be changed when we publish new version of ml-runtime-python

**Comment by marina-bjegovic** [comment]
What was done:
1. We now have RegressionEvaluator that inherits AbstractEvaluator (implemented in ml-runtime-python)
2. Data Scientist can use default methods from RegressionEvaluator or can override them
3. In this retail sample, we use default "evaluate" method and we override default "split" method.

File: recipes/python/retail/retail/evaluator.py [hunk 2]
```diff
@@ -14,33 +14,16 @@
...
-        metric = [{"name": "MAPE", "value": mape, "valueType": "double"},
-                  {"name": "MAE", "value": mae, "valueType": "double"},
-                  {"name": "RMSE", "value": rmse, "valueType": "double"}]
-
-        return metric
-
-
-
     def split(self, configProperties={}):
         #########################################
```
Comment (tumulurik) [inline recipes/python/retail/retail/evaluator.py]: Loading the data here and as well from runtime.
Left a comment on the PR for runtime - https://git.corp.adobe.com/ml/ml-runtime-python/pull/62

**Review by sathyarun18 — APPROVED** [review]
Looks good

**Review by tumulurik — APPROVED** [review]
LGTM

**Review by tatsuyawi — APPROVED** [review]

**Pull request merged**